### PR TITLE
Fixes box's power issues.

### PR DESCRIPTION
**Pull request by CRITAWAKETS**
## About The Pull Request

This PR fixes box's power issues. (dont mix shuttle powernets with station powernets)

## Why It's Good For The Game

box is playable now

## Changelog
:cl:
fix: Nanotrasen has fixed the power singularity inside of their Box-Pattern Mining Shuttle.
/:cl:

### CODE REVIEW

File: _maps/shuttles/mining_large.dmm [hunk 1]
```diff
@@ -69,7 +69,9 @@
 	},
 /obj/effect/turf_decal/bot_white,
 /turf/open/floor/plasteel/dark,
-/area/shuttle/mining/large)
+/area/shuttle/mining/large{
+	requires_power = 0
+	})
```
Comment (MarkSuckerberg) [inline _maps/shuttles/mining_large.dmm:74]:
```suggestion
/area/shuttle/mining)
```

**Comment by MarkSuckerberg** [comment]
screw it, let's go

**Comment by TheNeoGamer42** [comment]
oh wait did you actually just take the box mining shuttle and just remove its APC instead of making box use a different smaller shuttle?

**Comment by SynnGraffkin** [comment]
I'm still not entirely sold this will fix it.

**Pull request merged**